lib/api/index.js

```diff
@@ -359,14 +360,33 @@ app.post('/:installationId/migrate/:undo?', [
 
   try {
     if (req.params.undo) {
-      await jiraClient.devinfo.migration.undo();
+      /**
+       * Only call github/migrationUndo in prod. Undo will only be called in Jira if
+       * GITHUB_CONNECT_APP_IDENTIFIER is equal to com.github.integration.production
+       */
+      if (isProd()) {
+        await jiraClient.devinfo.migration.undo();
+        res.send(`Successfully called migration undo for ${installationId}`);
+      } else {
+        res.send('migrationUndo will only be called in prod');
```

Comment:
Looking for thoughts on what to send here/alternatives on how to handle. We don't want to send a response with `Successfully called migration undo for ${installationId}` unless in prod. Same goes for the complete response below.